src/types/resource.ts

```diff
@@ -12,7 +12,8 @@ export type ResourceType =
   | "bigquey"
```

Comment:
Woah! how did we miss this?
```suggestion
  | "bigquery"
```

Reply:
woahhhh LOL